ShopApp/ShopApp/View/LocationMapCell.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  LocationMapCell.swift
+//  ShopApp
+//
+//  Created by Andreea Sburlea on 10.06.2022.
+//
+
+import UIKit
+import MapKit
+
+class LocationMapCell: UITableViewCell {
+
+    @IBOutlet var mapView: MKMapView!
+
+    func configure(location: String) {
+        // Get location
+        let geoCoder = CLGeocoder()
+        print(location)
+        geoCoder.geocodeAddressString(location, completionHandler: { placemarks, error in
+            if let error = error {
+                print(error.localizedDescription)
+
+                return
+            }
+        if let placemarks = placemarks {
```

Comment:
This "if" should be aligned with the one above it and one empty space should exist in between them